airbyte-integrations/connectors/source-oracle/src/test/java/io/airbyte/integrations/source/oracle/OracleJdbcSourceAcceptanceTest.java

```diff
@@ -115,6 +131,107 @@ void cleanUpTables() throws SQLException {
       conn.close();
   }
 
+  @Override
+  protected List<AirbyteMessage> getTestMessages() {
+    return Lists.newArrayList(
+        new AirbyteMessage().withType(Type.RECORD)
+            .withRecord(new AirbyteRecordMessage().withStream(streamName)
+                .withNamespace(getDefaultNamespace())
+                .withData(Jsons.jsonNode(ImmutableMap
+                    .of(COL_ID, ID_VALUE_1,
+                        COL_NAME, "picard",
+                        COL_UPDATED_AT, "2004-10-19T00:00:00.0000Z")))),
+        new AirbyteMessage().withType(Type.RECORD)
+            .withRecord(new AirbyteRecordMessage().withStream(streamName)
+                .withNamespace(getDefaultNamespace())
+                .withData(Jsons.jsonNode(ImmutableMap
+                    .of(COL_ID, ID_VALUE_2,
+                        COL_NAME, "crusher",
+                        COL_UPDATED_AT,
+                        "2005-10-19T00:00:00.0000Z")))),
+        new AirbyteMessage().withType(Type.RECORD)
+            .withRecord(new AirbyteRecordMessage().withStream(streamName)
+                .withNamespace(getDefaultNamespace())
+                .withData(Jsons.jsonNode(ImmutableMap
+                    .of(COL_ID, ID_VALUE_3,
+                        COL_NAME, "vash",
+                        COL_UPDATED_AT, "2006-10-19T00:00:00.0000Z")))));
+  }
+
+  @Test
+  void testIncrementalTimestampCheckCursor() throws Exception {
```

Comment:
does this override the method from `JdbcSourceAcceptanceTest`? and if yes, why can't the original JdbcSourceAcceptanceTest method be modified? (same question for `testReadOneTableIncrementallyTwice`)

Reply:
Answered in the comment above